src/otomi.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * Note that running this script requires "--experimental-specifier-resolution=node" due to this being an ESM module
+ * https://nodejs.org/api/esm.html#esm_mandatory_file_extensions
+ * https://nodejs.org/api/esm.html#esm_customizing_esm_specifier_resolution_algorithm
+ * Run as:
+ *  node --experimental-specifier-resolution=node ./dist/otomi.js -- <args>
+ */
+
+import { lstatSync, readdirSync } from 'fs'
+import { CommandModule } from 'yargs'
+import { bootstrap, commands, defaultCommand } from './cmd'
+import { terminal } from './common/debug'
+import { ENV, LOG_LEVELS, parser } from './common/no-deps'
+import { otomi } from './common/setup'
+
+const debug = terminal('global')
+const terminalScale = 0.75
+if (!('OTOMI_IN_DOCKER' in process.env)) debug.exit(1, 'Please run this script using the `otomi` entry script')
+
+const envDirContent = readdirSync(ENV.DIR)
+if (envDirContent.length > 0) {
+  try {
+    let errorMessage = ''
+    if (!lstatSync(`${ENV.DIR}/.git`).isDirectory()) errorMessage += `\n${ENV.DIR}/.git is not a directory`
+    if (!lstatSync(`${ENV.DIR}/env`).isDirectory()) errorMessage += `\n${ENV.DIR}/env is not a directory`
+    if (!lstatSync(`${ENV.DIR}/env/charts`).isDirectory()) errorMessage += `\n${ENV.DIR}/env/charts is not a directory`
+    if (!lstatSync(`${ENV.DIR}/env/cluster.yaml`).isFile())
+      errorMessage += `\n${ENV.DIR}/env/cluster.yaml is not a file`
+    if (!lstatSync(`${ENV.DIR}/env/settings.yaml`).isFile())
+      errorMessage += `\n${ENV.DIR}/env/settings.yaml is not a file`
+    if (errorMessage.trim().length > 0) {
```

Comment:
I receive this error:

```
otomix kind -h
otomi/core:kind
  otomi:global:crit  It seems like '/home/sv/otomi/otomi-values-all/eks-dev' is not a valid values repo.
ENOENT: no such file or directory, lstat '/home/sv/otomi/otomi-values-all/eks-dev/.git' +0ms
```

Personally I would like the `ENV_DIR` to be a bit more permissive whether or not it is a Git repo, for local development the workflow kinda sucks (not blaming anyone btw), but I already made my point in another issue (https://github.com/redkubes/otomi-api/issues/176)

Reply:
I agree, I removed the check for .git - found that error as well (about an hour ago, just like you 😅 )

It'll no longer check for a .git dir